### PR TITLE
Update component to remove deprecated cfndsl syntax

### PR DESCRIPTION
**Pull request by Samseppiol**

### CODE REVIEW

File: sqs.cfndsl.rb [hunk 1]
```diff
@@ -4,7 +4,10 @@
   tags << { Key: 'Environment', Value: Ref(:EnvironmentName) }
   tags << { Key: 'EnvironmentType', Value: Ref(:EnvironmentType) }
 
+  extra_tags = external_parameters.fetch(:extra_tags, [])
```
Comment (Guslington) [inline sqs.cfndsl.rb]: default should be a hash cause we're looping over an expecting a `|key,value|`

File: sqs.cfndsl.rb [hunk 2]
```diff
@@ -4,7 +4,10 @@
...
   extra_tags.each { |key,value| tags << { Key: key, Value: value } } if defined? extra_tags
```
Comment (Guslington) [inline sqs.cfndsl.rb]: no need for the if defined cause it's being set above with a default value

**Review by Guslington — CHANGES_REQUESTED** [review]
couple of changes, see comments

File: sqs.cfndsl.rb (hunk 1 above)
Comment (Samseppiol) [reply sqs.cfndsl.rb]: copy paste fail :S

**Review by Guslington — APPROVED** [review]
lgtm

**Pull request merged**